tests/core/test_document.py

```diff
@@ -0,0 +1,23 @@
+from ragna.core import DocxDocumentHandler, LocalDocument
+
+
+def get_docx_document(tmp_path, docx_text):
+    import docx
```

Comment:
Nit: you can put this at the top level of the module. Here we are in a development environment and this is always available.

Reply:
I moved the import to the top level